packages/cmrjs/src/cmr-utils.js

```diff
@@ -389,8 +469,11 @@ function mapCNMTypeToCMRType(type, urlType = 'distribution', useDirectS3Type = f
     metadata: 'EXTENDED METADATA',
     qa: 'EXTENDED METADATA',
   };
-  const mappedType = mapping[type] || 'GET DATA';
 
+  let mappedType = 'GET DATA';
```

Comment:
This was required as mapping[type: string | undefined] isn't always valid with type safety

Reply:
Good catch!   That was true when I updated the typedoc, but no longer.   4c48963bb03